File: random.pl
#!/usr/bin/env perl
use v5.10.0;
use warnings;
use strict;

# return a random number between min and max

my $lower_limit = 0;
my $upper_limit = 1000;

my $random_number = int(rand($upper_limit-$lower_limit)) + $lower_limit;

print $random_number, "\n";

File: random.rb
# Generate random number of range
# To run this program ruby random.rb
def random_number
  puts "Enter the lower limit"
  lower_limit = gets.chomp.to_i
  puts "Enter the heighest limit"
  upper_limit = gets.chomp.to_i
  num = rand(lower_limit..upper_limit)
  puts "Your random number is #{num}"
end

random_number
